example/javascriptlib/module/5-resources/build.mill

```diff
@@ -0,0 +1,30 @@
+package build
+
+import mill._, javascriptlib._
+import ujson._
+
+object foo extends TypeScriptModule {
+  object test extends TypeScriptTests with TestModule.Jest {
+    def mkENV = super.mkENV() + ("OTHER_FILES" -> (compile()._2.path / "other-files").toString)
```

Comment:
Why is this not `def otherFiles = Task.Source(millSourcePath / "other-files")` like it is in the java/scala/kotlin examples?